Use the raintpl package provided in Debian instead of the hard copy of
its source code (which is already removed from the DFSG repackaged
archive)
--- a/index.php
+++ b/index.php
@@ -52,7 +52,7 @@
         ."- https://shaarli.readthedocs.io/en/master/Download-and-Installation/";
     exit;
 }
-require_once 'inc/rain.tpl.class.php';
+require_once '/usr/share/raintpl/inc/rain.tpl.class.php';
 require_once __DIR__ . '/vendor/autoload.php';
 
 // Shaarli library
